src/components/FloatingBar/FloatingBar.css

```diff
@@ -60,3 +60,9 @@
   margin-right: 84px !important;
   position: relative !important;
 }
+
+@media (max-width: 374px) {
```

Comment:
I'd do the opposite here. `display: none` as default and add a min-width media query so it has a mobile-first approach.